main.go

```diff
@@ -1545,6 +1545,10 @@ func postProcessFiles(inFiles chan sumFileJob, wg *sync.WaitGroup, opts options)
 						}
 						continue
 					}
+					if opts.DeleteLocalFileAfterUpload {
+						l.Verboseln("removing", j.Path)
```

Comment:
The info level would be better to explicitly inform the user that the file is being removed

Reply:
@orgrim :+1: done